travis/ubuntu-compile-nsenter.sh

```diff
@@ -17,7 +17,9 @@ set -e
 
 sudo apt-get update
 sudo apt-get install libncurses5-dev libslang2-dev gettext zlib1g-dev libselinux1-dev debhelper lsb-release pkg-config po-debconf autoconf automake autopoint libtool
-mkdir .tmp || true
-wget https://www.kernel.org/pub/linux/utils/util-linux/v2.30/util-linux-2.30.2.tar.gz -qO - | tar -xz -C .tmp/
-cd .tmp/util-linux-2.30.2 && ./autogen.sh && ./configure && make nsenter
+mkdir $HOME/nsenter || true
+if [ ! -d $HOME/nsenster/util-linux-2.30.2 ]; then
```

Comment:
Would it make sense to externalize the version into a var? Otherwise, updating the version will be a PITA :)

Reply:
I will keep this one as the version is also used in travis.yml and here it has two forms :/